rust/kcl-lib/tests/circular_pattern3d_a_pattern/ops.snap

```diff
@@ -53,10 +53,15 @@ description: Operations executed circular_pattern3d_a_pattern.kcl
     "name": "patternLinear3d",
     "unlabeledArg": {
       "value": {
-        "type": "Solid",
-        "value": {
-          "artifactId": "[uuid]"
-        }
+        "type": "Array",
+        "value": [
+          {
+            "type": "Solid",
+            "value": {
+              "artifactId": "[uuid]"
```

Comment:
Suspicious that these changed, but I don't think the UI knows how to deal with patterns yet. So I guess it's fine.